Missionframework/modules/05_garrison/fnc/fn_garrison_spawnSectorVehicle.sqf

```diff
@@ -0,0 +1,75 @@
+/*
+    KPLIB_fnc_garrison_spawnSectorVehicle
+
+    File: fn_garrison_spawnSectorVehicle.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-10-21
+    Last Update: 2018-10-23
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Spawns a vehicle with full crew of given side at given sector.
+
+    Parameter(s):
+        _sector         - Markername of the sector          [STRING, defaults to ""]
+        _vehicle        - Classname of the vehicle to spawn [STRING, defaults to ""]
+        _side           - Side of the vehicle               [SIDE, defaults to KPLIB_preset_sideEnemy]
+        _kind           - Kind of the vehicle (light/heavy) [STRING, defaults to "light"]
+
+    Returns:
+        Spawned vehicle [OBJECT]
+*/
+
+params [
+    ["_sector", "", [""]],
+    ["_vehicle", "", [""]],
+    ["_side", KPLIB_preset_sideEnemy, [sideEmpty]],
+    ["_kind", "light", [""]]
+];
+
+// Exit if no or invalid sector or classname was given
+if (_sector == "") exitWith {objNull};
+if (_vehicle == "") exitWith {objNull};
+if !(_sector in KPLIB_sectors_all) exitWith {objNull};
+
+// Initialize local variables
+private _vehicleArray = [];
+private _sectorPos = getMarkerPos _sector;
+private _spawnPos = [_sectorPos] call KPLIB_fnc_garrison_getVehSpawnPos;
+private _activeGarrisonRef = KPLIB_garrison_active select (KPLIB_garrison_active findIf {(_x select 0) == _sector});
+
+/* NOTE
+    This works totally fine and also adds e.g. a CSAT vehicle classname as Blufor side, if ordered.
+    The "problem" is, that it has the normal config crew, so it's manned by CSAT soldiers which belong to Blufor side.
+    Maybe we have to replace it with an own function, especially as some AAF vehicles (resistance side) are also in the blufor preset.
+*/
+// Spawn vehicle
+_vehicleArray = [_spawnPos, random 360, _vehicle, _side] call BIS_fnc_spawnvehicle;
```

Comment:
Same concern as infantry, we already have `common_spawnVehicle` function, so maybe we could utilize that + `common_createCrew` function.

Reply:
Agree, will do. 👍